frameworklauncher/README.md

```diff
@@ -77,6 +77,6 @@ Windows cmd line:
 GNU/Linux cmd line:
 
     ./dist/start.sh
-    
+
 ## User Manual
 See [User Manual](doc/USERMANUAL.md) to learn how to use Launcher Service to Launch Framework.
```

Comment:
Only space changes, suggest to revert this file.

Reply:
Suggest to unify the space style in Launcher.

---
In reply to: [166509161](https://github.com/Microsoft/pai/pull/309#discussion_r166509161) [](ancestors = 166509161)